pkgs/tools/audio/goxlr-utility/default.nix

```diff
@@ -0,0 +1,65 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, installShellFiles
+, pkg-config
+, libpulseaudio
+, dbus
+, speechd
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "goxlr-utility";
+  version = "v0.12.6";
+
+  src = fetchFromGitHub {
+    owner = "GoXLR-on-Linux";
+    repo = pname;
```

Comment:
Many people dislike this not really necessary link between `pname` and `repo`.

Reply:
Assuming that you mean that we'd prefer writing the repo name explicitly? Changed it now.